k8s/deploy-storageos/upgrade-helper/README.md

```diff
@@ -0,0 +1,16 @@
+# StorageOS Upgrade Helper
+
+This section aims to help users with the StorageOS upgrade from any version
```

Comment:
Maybe call it a script rather than a section? This is a nitpick

Reply:
done